packages/logs/src/domain/logsCollection/logger/loggerCollection.ts

```diff
@@ -51,3 +57,10 @@ export function isAuthorized(status: StatusType, handlerType: HandlerType, logge
     STATUS_PRIORITIES[status] >= STATUS_PRIORITIES[logger.getLevel()] && includes(sanitizedHandlerType, handlerType)
   )
 }
+
+function sendToConsole(logsMessage: LogsMessage, logger: Logger) {
+  if (isAuthorized(logsMessage.status, HandlerType.console, logger)) {
+    const api = objectHasValue(ConsoleApiName, logsMessage.status) ? logsMessage.status : ConsoleApiName.log
+    display[api](logsMessage.message, combine(logger.getContext(), logsMessage.context))
+  }
+}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** `sendToConsole` naming seems a bit odd now.
What about inlining it and maybe extract a `display(status, message, context)` to `display.ts` instead?
‏